<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcomeWidget</class>
 <widget class="QWidget" name="welcomeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblGif">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widgetDateTime" native="true">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>315</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>315</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>315</width>
     <height>100</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 84);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="lblDate">
      <property name="styleSheet">
       <string notr="true">font: 25pt &quot;微软雅黑&quot;;
color: rgb(236, 236, 236);
background-color: rgba(0, 0, 0, 0);</string>
      </property>
      <property name="text">
       <string>2020年04月15日</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblTime">
      <property name="styleSheet">
       <string notr="true">font: 25pt &quot;黑体&quot;;
color: rgb(236, 236, 236);
background-color: rgba(0, 0, 0, 0);</string>
      </property>
      <property name="text">
       <string>11:11:11</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetVersion" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>511</width>
     <height>53</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 84);</string>
   </property>
   <widget class="QLabel" name="lblVersion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>7</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25pt &quot;微软雅黑&quot;;
color: rgb(236, 236, 236);
background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string>看番神器Pro V1.X.X(2020.XX.XX)</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>379</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>315</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1010</width>
     <height>1010</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 84);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="lblHello">
      <property name="styleSheet">
       <string notr="true">font: 23pt &quot;微软雅黑&quot;;
color: rgb(236, 236, 236);
background-color: rgba(0, 0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>下午好 Tanyiqu！</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblHelloWord">
      <property name="styleSheet">
       <string notr="true">font: 23pt &quot;微软雅黑&quot;;
color: rgb(236, 236, 236);
background-color: rgba(0, 0, 0, 0);</string>
      </property>
      <property name="text">
       <string>今天想要看点什么呢？
我这里什么都有哦！</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
